src/Bundle/spec/Factory/ExtenderGridFactorySpec.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\GridBundle\Factory;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\GridBundle\Grid\GridInterface;
+use Sylius\Bundle\GridBundle\Registry\GridRegistryInterface;
+use Sylius\Component\Grid\Configuration\GridConfigurationExtenderInterface;
+use Sylius\Component\Grid\Definition\Grid;
+use Sylius\Component\Grid\Factory\GridFactoryInterface;
+
+final class ExtenderGridFactorySpec extends ObjectBehavior
```

Comment:
This is not necessary here, but for the future it would be good to add new unit tests in PHPUnit, not in PHPSpec 🖖🏻